Bio/SeqIO/Interfaces.py

```diff
@@ -91,9 +95,13 @@ def __iter__(self):
         """
         return self
 
+    @abstractmethod
+    def parse(self, handle: IO[Any]) -> Generator[SeqRecord, None, None]:
+        """Start parsing the file, and return a SeqRecord generator."""
+
```

Comment:
Why do you add an abstract ``.parse`` method?

Reply:
Because there is code that assumes all `SequenceIterator`s implement such a method (and indeed, all sub-classes do implement `parse`). This was the best of the available solutions and is good documentation

Reply:
Thanks. This is clearer. I'd be happy to see that as a standalone PR.

Reply:
https://github.com/biopython/biopython/pull/3176 :+1: